dependencies.yaml

```
@@ -565,7 +565,7 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - cachetools
-          - &numba numba>=0.57
+          - &numba_cuda numba-cuda>=0.0.13
```

IIUC this is assigning to `numba_cuda` - is that the best form for a name to avoid having a dash in a name, or for consistency with the package name should I change this back to:
```suggestion
          - &numba-cuda numba-cuda>=0.0.13
```

with corresponding change below?

---

I personally don't like using just plain package names as the anchor names, because it makes it harder to figure out  what to look/search for. I prefer things like `numba-cuda-dep`, so that the YAML anchor is distinct from the package name.

That might just be me, though, so I would advise you to follow your best judgement.

---

One other thing to keep in mind is that a YAML anchor can represent arbitrary chunks of YAML. It is not necessarily just one dependency. A naming formalism might help us with this ambiguity.

---

I have no capacity for judgement here (I'm just cargo-cult changing things) so I'll go with your suggestion :-)

```suggestion
          - &numba-cuda-dep numba-cuda>=0.0.13
```